validator/webui/index.html

```diff
@@ -91,15 +68,28 @@
        the cursor around. See https://codemirror.net/doc/manual.html when
        adding additional features. -->
   <dom-module id="amphtml-editor">
+    <style scope="amphtml-editor">
+      .CodeMirror-cursor {
+        border-left: 3px solid black;
+        border-right: none;
+        width: 3;
+      }
+    </style>
     <template>
       <paper-material elevation="1">
         <textarea id="code" name="code" rows="100"
-                  style="width:100%;height:300px"></textarea>
+                  style="width:100%"></textarea>
```

Comment:
Cool, turns out I can just delete the formatting fields, even rows, since CodeMirror jiggles it anyway once it takes over the dom node.